package.json

```diff
@@ -2,51 +2,56 @@
-        "@babel/plugin-transform-runtime": "^7.17.0",
-        "@babel/preset-env": "^7.16.8",
-        "@typescript-eslint/eslint-plugin": "^6.18.1",
-        "@typescript-eslint/parser": "^6.18.1",
+        "@babel/plugin-transform-class-properties": "^7.24.1",
+        "@babel/plugin-transform-runtime": "^7.24.3",
+        "@babel/preset-env": "^7.24.2",
+        "@typescript-eslint/parser": "^7.7.0",
         "@vitejs/plugin-vue": "^5.0.4",
-        "@vitest/coverage-v8": "^1.4.0",
+        "@vitest/coverage-v8": "^1.5.0",
         "@vue/test-utils": "^2.4.5",
-        "babel-loader": "^8.2.3",
+        "babel-loader": "^9.1.3",
         "clean-webpack-plugin": "^4.0.0",
-        "copy-webpack-plugin": "^10.2.0",
+        "copy-webpack-plugin": "^12.0.2",
         "cross-env": "^7.0.3",
-        "cross-fetch": "^3.1.5",
-        "css-loader": "^6.5.1",
-        "eslint": "^8.18.0",
-        "eslint-plugin-vue": "^9.20.0",
+        "cross-fetch": "^4.0.0",
+        "css-loader": "^7.1.1",
+        "eslint": "^9.0.0",
+        "eslint-plugin-vue": "^9.25.0",
         "file-loader": "^6.2.0",
-        "html-loader": "^3.1.0",
-        "html-webpack-plugin": "^5.5.0",
+        "html-loader": "^5.0.0",
+        "html-webpack-plugin": "^5.6.0",
         "jsdom": "^24.0.0",
-        "mini-css-extract-plugin": "^2.5.1",
-        "nodemon": "^3.0.2",
-        "postcss": "^8.4.14",
-        "postcss-loader": "^6.2.1",
-        "postcss-preset-env": "^7.2.3",
+        "mini-css-extract-plugin": "^2.8.1",
+        "nodemon": "^3.1.0",
+        "postcss": "^8.4.38",
+        "postcss-loader": "^8.1.1",
+        "postcss-preset-env": "^9.5.5",
         "raw-loader": "^4.0.2",
-        "sass": "^1.70.0",
-        "sass-loader": "^12.4.0",
-        "style-loader": "^3.3.1",
-        "stylelint": "^14.2.0",
-        "stylelint-config-standard": "^24.0.0",
-        "stylelint-webpack-plugin": "^3.1.1",
+        "sass": "^1.75.0",
+        "sass-loader": "^14.2.0",
+        "style-loader": "^4.0.0",
+        "stylelint": "^16.3.1",
+        "stylelint-config-standard": "^36.0.0",
+        "stylelint-webpack-plugin": "^5.0.0",
         "text-loader": "^0.0.1",
         "ts-loader": "^9.5.1",
-        "typescript": "^5.3.3",
-        "vitest": "^1.4.0",
-        "vue-loader": "^17.2.2",
-        "vue-template-compiler": "^2.7.14",
-        "vue-tsc": "^2.0.6",
-        "webpack": "^5.77.0",
-        "webpack-cli": "^4.9.1",
-        "webpack-dev-server": "^4.7.3",
-        "webpack-merge": "^5.8.0"
+        "typescript": "^5.4.5",
+        "typescript-eslint": "^7.7.0",
+        "vitest": "^1.5.0",
+        "vue-loader": "^17.4.2",
+        "vue-template-compiler": "^2.7.16",
+        "vue-tsc": "^2.0.13",
+        "webpack": "^5.91.0",
+        "webpack-cli": "^5.1.4",
+        "webpack-dev-server": "^5.0.4",
+        "webpack-merge": "^5.10.0"
+    },
+    "resolutions": {
+        "eslint": "^8.56.0",
+        "vite": "5.0.0",
+        "vue": "^3.2.25"
```

Comment:
Thanks. I've only just learned what peer dependencies are. I found [this link](https://www.johno.com/using-peer-dependencies) helpful.

Still being on v1 yarn (lol) it's possible this is too noisy / incorrect, but with the latest, I'm getting:
```js
warning "arches-dev-dependencies > @babel/eslint-parser@7.24.1" has incorrect peer dependency "eslint@^7.5.0 || ^8.0.0".
warning "arches-dev-dependencies > @typescript-eslint/parser@7.7.0" has incorrect peer dependency "eslint@^8.56.0".
warning "arches-dev-dependencies > @vitejs/plugin-vue@5.0.4" has unmet peer dependency "vite@^5.0.0".
warning "arches-dev-dependencies > typescript-eslint@7.7.0" has incorrect peer dependency "eslint@^8.56.0".
warning "arches-dev-dependencies > typescript-eslint > @typescript-eslint/eslint-plugin@7.7.0" has incorrect peer dependency "eslint@^8.56.0".
warning "arches-dev-dependencies > typescript-eslint > @typescript-eslint/utils@7.7.0" has incorrect peer dependency "eslint@^8.56.0".
warning "arches-dev-dependencies > typescript-eslint > @typescript-eslint/eslint-plugin > @typescript-eslint/type-utils@7.7.0" has incorrect peer dependency "eslint@^8.56.0".
```

We can probably ignore the "incorrect" warnings as noise, but the one "unmet" warning says we need to resolve to vite 5.0.0.

```
$ yarn list vite
yarn list v1.22.19
warning Filtering by arguments is deprecated. Please use the pattern option instead.
└─ vite@5.2.9
✨  Done in 0.20s.
```

Thoughts? Possible?

Reply:
Yeah those resolution statement don't appear to do anything 😞 

IMO this is all just noise that will hopefully go away when we update/change package managers. I'm unsure if there's any actions we could/should do to resolve this. What do you think?

Reply:
Looks like it's a busy time for tooling with the eslint 9 release.

- `@babel/eslint-parser` somewhere between 7.24.1 > 7.24.5 adds eslint 9 as acceptable peer, so let's take that
- `@typescript-eslint`: we [need](https://github.com/typescript-eslint/typescript-eslint/issues/8211) to be on v8-alpha-10 or newer
- `@vitejs/plugin-vue@5.0.4` wants us to take vue as a dep, which makes sense

See diff in next comment that worked for me